src/wazuh_modules/wm_ciscat.c

```diff
@@ -167,6 +167,26 @@ void* wm_ciscat_main(wm_ciscat *ciscat) {
         ciscat->flags.error = 1;
     }
 
+    #ifdef WIN32
+        if (ciscat->ciscat_binary == '\0') {
```

Comment:
`ciscat_binary` is `char *`. Thus, we should compare it with a pointer.
```suggestion
        if (ciscat->ciscat_binary == NULL) {
```

Reply:
> `ciscat_binary` is `char *`. Thus, we should compare it with a pointer.
> 
> ```
> wazuh_modules/wm_ciscat.c:180:35: warning: comparison between pointer and zero character constant [-Wpointer-compare]
> ```

If we change to compare it with a pointer. The previous user needs to add the configuration  `<ciscat_binary>` in the `/var/ossec/etc/ossec.conf` manully, otherwise there is an error like this

![image](https://user-images.githubusercontent.com/6824093/160048135-3e932c9a-9c1e-43a9-9aca-933ec456aaad.png)


If we compare it with `\0`,  The above error will no longer appear. @vikman90